conf/app.ini

```diff
@@ -20,6 +20,26 @@ MAX_CREATION_LIMIT = -1
 ; Patch test queue length, make it as large as possible
 PULL_REQUEST_QUEUE_LENGTH = 10000
 
+[editor]
+; List of file extensions that should have line wraps in the CodeMirror editor
+; Separate extensions with a comma. To line wrap files w/o extension, just put a comma
+LINE_WRAP_EXTENSIONS = .txt,.md,.markdown,.mdown,.mkd,,
```

Comment:
Actually it as meaning, as I wrote in the comment before it. I found that usually when a person has a file without a file extension, which a comma that succeeds nothing indicates, it is just a text file, usually a README file or similar. I usually wordwrap those.